src/helpers/rgba.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import rgb from './rgb'
+
+type RgbaColor = {
+  red: number,
+  green: number,
+  blue: number,
+  alpha: number,
+}
+
+/**
+ * Returns a string value for the color. The returned result is the smallest possible rgba or hex notation.
+ *
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   background: rgba(255, 205, 100, 180),
+ *   background: rgba({ red: 255, green: 205, blue: 100, alpha: 180 }),
+ *   background: rgba(255, 205, 100, 255),
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.div`
+ *   background: ${rgba(255, 205, 100, 180)};
+ *   background: ${rgba({ red: 255, green: 205, blue: 100, alpha: 180 })};
+ *   background: ${rgba(255, 205, 100, 255)};
+ * `
+ *
+ * // CSS in JS Output
+ *
+ * element {
+ *   background: "rgba(255,205,100,180)";
+ *   background: "rgba(255,205,100,180)";
+ *   background: "#ffcd64";
+ * }
+ */
+function rgba(value: RgbaColor | number, green?: number, blue?: number, alpha?: number): string {
+  if (typeof value === 'number' &&
+      typeof green === 'number' &&
+      typeof blue === 'number' &&
+      typeof alpha === 'number') {
```

Comment:
Sound reasonable to me to let errors bubble up via the browser. It's a helper library and not validating all of your CSS 👍 

I tried to think about when I used rgba or how I would use rgba without providing the alpha. Especially in transition animations as @bhough mentioned it never would be `null` or `undefined` and if it would, it would be a bug in my calculations.

That's what I'm mostly worried about. If we default back to 1, people might miss bugs. Can you think of an transition animation calculation where alpha would be `null` or `undefined`?